ndbench-core/src/main/java/com/netflix/ndbench/core/config/IConfiguration.java

```diff
@@ -89,7 +96,25 @@ default int getNumReaders() {
     @DefaultValue("100")
     int getReadRateLimit();
 
+
     @DefaultValue("100")
+    @PropertyName(name=WRITE_RATE_LIMIT)
     int getWriteRateLimit();
 
+
+    @DefaultValue("false")
+    boolean isAutoTuneEnabled();
+
+
+    @DefaultValue("60")
```

Comment:
Hi. I have committed / pushed the changes in nfndbench that depend on the changes to the OSS project and have added you as a reviewer.